packages/teams-js/src/private/teams.ts

```diff
@@ -82,4 +85,77 @@ export namespace teams {
   export function isSupported(): boolean {
     return runtime.supports.teams ? true : false;
   }
+
+  export namespace fullTrust {
+    export namespace joinedTeams {
+      /**
+       * @hidden
+       * Hide from docs
+       * ------
+       * Allows an app to retrieve information of all user joined teams
+       *
+       * @param teamInstanceParameters - OPTIONAL Flags that specify whether to scope call to favorite teams
+       * @returns Promise resolved containing information about the user joined teams or rejected with error
+       */
+      export function getUserJoinedTeams(
+        teamInstanceParameters?: TeamInstanceParameters,
+      ): Promise<UserJoinedTeamsInformation> {
+        return new Promise<UserJoinedTeamsInformation>(resolve => {
+          ensureInitialized();
+          if (!isSupported()) {
+            throw errorNotSupportedOnPlatform;
+          }
+
+          if (
+            (GlobalVars.hostClientType === HostClientType.android ||
+              GlobalVars.hostClientType === HostClientType.teamsRoomsAndroid ||
+              GlobalVars.hostClientType === HostClientType.teamsPhones ||
+              GlobalVars.hostClientType === HostClientType.teamsDisplays) &&
+            !isCurrentSDKVersionAtLeast(getUserJoinedTeamsSupportedAndroidClientVersion)
+          ) {
+            const oldPlatformError: SdkError = { errorCode: ErrorCode.OLD_PLATFORM };
+            throw new Error(JSON.stringify(oldPlatformError));
+          }
+
+          resolve(sendAndUnwrap('getUserJoinedTeams', teamInstanceParameters));
+        });
+      }
+
```

Comment:
Can you please add this above `isSupported`:

      /**
       * @hidden
       * Hide from docs
       * ------
       * Checks if teams.fullTrust.joinedTeams capability is supported by the host
       * @returns true if the teams.fullTrust.joinedTeams capability is enabled in
       * runtime.supports.teams.fullTrust.joinedTeams and false if it is disabled
       */